packages/lit-node-client/src/lib/lit-node-client.ts

```diff
@@ -234,6 +237,7 @@ export class LitNodeClient {
       authSig: params.authSig,
       jsParams: convertLitActionsParams(params.jsParams),
       requestId: Math.random().toString(16).slice(2),
+      singleNode: params.singleNode ?? false,
```

Comment:
not sure if this is the right place to put the default of false?

Reply:
thinking about this more...  i think singleNode and a boolean is wrong.  i think instead we should let them specify how many nodes it should run on.  there are def cases where someone might wanna run something on just 3 nodes instead of 1

Reply:
I've removed `singleNode:boolean` and added `targerNodeRange?: number` instead:
```
targetNodeRange: params.targetNodeRange ?? 0,
```

then we can do something like:
```js
  // -- only run on a single node
  if (targetNodeRange && targetNodeRange === 1) {
    res = await this.runOnSingleNode(params);
  
  // -- run on multiple nodes
  } else if (targetNodeRange && targetNodeRange > 1) {
    throw new Error('this.runOnMultipleNodes(params) not implemented yet');
  
  // -- run on all nodes (default)
  } else {
    ```

Reply:
🔥